xml/soc-operations-self-sign-cert.xml

```diff
@@ -0,0 +1,784 @@
+    </step>
+    <step>
+      <para>
+        In order to set up TXT record in bind DNS, edit the zone file so
+        it looks like the following example:
+      </para>
+      <screen>
+        yourdomain.com.     IN NS           admin.yourdomain.com.
+        _acme-challenge.yourdomain.com. IN TXT "xxxx...your TXT value string here"
+      </screen>
+    </step>
+    <step>
+      <para>
+        Restart your named services for <literal>bind</literal>.
+      </para>
+    </step>
+    <step>
+      <para>
+        Issue the acme-challenge verification of the previous step with the
+        following command:
+      </para>
+      <screen>
+        # acme.sh --renew -d yourdomain.com
+      </screen>
+    </step>
+    <step>
+      <para>
+        If the DNS validation is okay, <filename>acme.sh</filename> issues a
+        wildcard certificate and displays the certificate and private-key path.
+        For example:
+      </para>
+      <screen>
+        Your cert is in:  /root/.acme.sh/susedojo.com/susedojo.com.cer  
+        Your cert key is in:  /root/.acme.sh/susedojo.com/susedojo.com.key  
+        v2 chain.
+        The intermediate CA cert is in:  /root/.acme.sh/susedojo.com/ca.cer  
+        And the full chain certs is in:  /root/.acme.sh/susedojo.com/fullchain.cer  
+        _on_issue_success
+      </screen>
+    </step>
+    <step>
+      <para>
+        Notice the location of your certificate and key. These are now
+        ready to be used by OpenStack Cloud.
+      </para>
+    </step>
+    <step>
+      <para>
+        To automate the process of setting up the TXT record in your DNS
+        servers and prepare it for automated validation, there is a file
+        called <filename>account.conf</filename> holds account information
+        for the DNS Provider. After exporting the authentication variables,
+        it stores them automatically after the command is executed for later use.
+        To issue your wildcard certificate, the command without optional settings is:
+      </para>
+      <screen>
+        # export LUA_Key=”your_API_token_from_account”
+        # export LUA_Email=”cameron@yourdomain.com”
+        # acme.sh -d yourdomain.com -d *.yourdomain.com --dns dns_lua
+      </screen>
+    </step>
+    <step>
+      <para>
+       You can now view your DNS records and you will see a new TXT record
+       available. When its finished and the DNS validation is okay,
+       <filename>acme.sh</filename> issue your wildcard certificate and displays
+       your certificate and private-key path just as before.
+      </para>
+      <screen>
+        Your cert is in:  /root/.acme.sh/susedojo.com/susedojo.com.cer  
+        Your cert key is in:  /root/.acme.sh/susedojo.com/susedojo.com.key  
+        v2 chain.
+        The intermediate CA cert is in:  /root/.acme.sh/susedojo.com/ca.cer  
+        And the full chain certs is in:  /root/.acme.sh/susedojo.com/fullchain.cer  
+        _on_issue_success
+      </screen>
```

Comment:
same question about with `v2 chain` and `_on_issue_success` should they be at the end of the previous lines?